ui/src/views/Releases/ListReleases/index.jsx

```diff
@@ -337,11 +431,19 @@ function ListReleases(props) {
 
         const ret = clone(r);
 
-        if (result.sc) {
-          ret.scheduledChange = buildScheduledChange(result.sc);
+        if (r.api_version === 1) {
+          if (result.sc) {
+            ret.scheduledChange = buildScheduledChange(result.sc);
+          } else {
+            ret.read_only = !r.read_only;
+            ret.data_version = result.new_data_version;
+          }
+        } else if (result['.'].sc_id) {
+          ret.scheduledChange = buildScheduledChangeV2(ret, [result['.']]);
+          ret.scheduledChange.read_only = !r.read_only;
         } else {
           ret.read_only = !r.read_only;
-          ret.data_version = result.new_data_version;
+          ret.data_version = result['.'];
```

Comment:
What does a property field `.` mean?

Reply:
Ah, the new API returns responses that have either data versions or scheduled change information for 1 or more parts of the Release. `.` refers to the top level of the Release. Perhaps this should be a constant?

Reply:
Sure, I think that works. A descriptive field name would be more readable than a dot :)